src/librawspeed.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@DEST_DIR@
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: librawspeed
+Description: A fast camera raw file decoder (https://github.com/darkroom-org/rawspeed).
+Version: 3.1
```

Comment:
As you might have seen in the tag description, the `3.1`,`3.0` are *NOT* releases.
The last release was `1.0` (git master), and there was `2.0` somewhere in between.
The `4.0` is **not** quite here yet.

Reply:
Ok, well I'll put it at 4.0 for my next attempt then, it would be cool if we could have a top level variable set in the CMAKE project which could then be injected into this file, and any others that need version numbers in the future. Maybe I'll do a separate PR for that.